serverless/pytorch/shiyinzhang/iog/nuclio/model_handler.py

```diff
@@ -50,10 +50,10 @@ def handle(self, image, bbox, pos_points, neg_points, threshold):
             # extract a crop with padding from the image
             crop_padding = 30
             crop_bbox = [
-                max(bbox[0] - crop_padding, 0),
-                max(bbox[1] - crop_padding, 0),
-                min(bbox[2] + crop_padding, image.width - 1),
-                min(bbox[3] + crop_padding, image.height - 1)
+                max(bbox[0][0] - crop_padding, 0),
```

Comment:
Is it unified with some other input parameters? Usually a bounding box is 4 coordinates [x0, y0, x1, y1]. I'm OK with any solution if there is a common ground behind the decision.

Reply:
Also it will be necessary to redeploy the function with the change. Again, I'm fine with that we we have strong reasons to unify different interfaces.

Reply:
![Screenshot from 2021-07-20 09-57-18](https://user-images.githubusercontent.com/40690378/126275775-b3cbd6a7-c517-47f4-b7b7-2d2ee0919b45.png)

Done like for points. 
We anyway need to re-deploy all the functions after adding ``min_neg_points``